config/environments/production.rb

```diff
@@ -64,6 +64,7 @@ module OptimizerWrapper
       demo: DEMO,
       vroom: VROOM,
       ortools: ORTOOLS,
+      unconstrainted_initialization: UNCONSTRAINTED_INITIALIZATION,
```

Comment:
The initialization of the variable is missing I think

`UNCONSTRAINTED_INITIALIZATION = Wrappers::UnconstraintedInitialization.new(tmp: TMP_DIR)`